diff --git a/node_modules/@radix-ui/react-scroll-area/dist/index.mjs b/node_modules/@radix-ui/react-scroll-area/dist/index.mjs
index 2194c76..f93653a 100644
--- a/node_modules/@radix-ui/react-scroll-area/dist/index.mjs
+++ b/node_modules/@radix-ui/react-scroll-area/dist/index.mjs
@@ -527,6 +527,10 @@ var ScrollAreaScrollbarImpl = React2.forwardRef((props, forwardedRef) => {
             const mainPointer = 0;
             if (event.button === mainPointer) {
               const element = event.target;
+              /* patch_start */
+              const isFirefox = navigator.userAgent.toLowerCase().includes("firefox");
+              if (isFirefox) document.body.style.pointerEvents = "none";
+              /* patch_end */
               element.setPointerCapture(event.pointerId);
               rectRef.current = scrollbar.getBoundingClientRect();
               prevWebkitUserSelectRef.current = document.body.style.webkitUserSelect;
@@ -541,6 +545,10 @@ var ScrollAreaScrollbarImpl = React2.forwardRef((props, forwardedRef) => {
             if (element.hasPointerCapture(event.pointerId)) {
               element.releasePointerCapture(event.pointerId);
             }
+            /* patch_start */
+            const isFirefox = navigator.userAgent.toLowerCase().includes("firefox");
+            if (isFirefox) document.body.style.pointerEvents = "auto";
+            /* patch_end */
             document.body.style.webkitUserSelect = prevWebkitUserSelectRef.current;
             if (context.viewport) context.viewport.style.scrollBehavior = "";
             rectRef.current = null;
