diff --git a/node_modules/@radix-ui/react-slider/dist/index.mjs b/node_modules/@radix-ui/react-slider/dist/index.mjs
index 4182c6a..29092bc 100644
--- a/node_modules/@radix-ui/react-slider/dist/index.mjs
+++ b/node_modules/@radix-ui/react-slider/dist/index.mjs
@@ -311,6 +311,10 @@ var SliderImpl = React.forwardRef(
         }),
         onPointerDown: composeEventHandlers(props.onPointerDown, (event) => {
           const target = event.target;
+          /* patch_start */
+          const isFirefox = navigator.userAgent.toLowerCase().includes("firefox");
+          if (isFirefox) document.body.style.pointerEvents = "none";
+          /* patch_end */
           target.setPointerCapture(event.pointerId);
           event.preventDefault();
           if (context.thumbs.has(target)) {
@@ -329,6 +333,10 @@ var SliderImpl = React.forwardRef(
             target.releasePointerCapture(event.pointerId);
             onSlideEnd(event);
           }
+          /* patch_start */
+          const isFirefox = navigator.userAgent.toLowerCase().includes("firefox");
+          if (isFirefox) document.body.style.pointerEvents = "auto";
+          /* patch_end */
         })
       }
     );
